tableauserverclient/server/endpoint/groups_endpoint.py

```diff
@@ -25,15 +25,22 @@ def get(self, req_options=None):
     # Gets all users in a given group
     @api(version="2.0")
     def populate_users(self, group_item, req_options=None):
+        from .. import Pager
         if not group_item.id:
             error = "Group item missing ID. Group must be retrieved from server first."
             raise MissingRequiredFieldError(error)
+
+        # TODO should this be a list or a Pager directly?
+        all_users = list(Pager(lambda options: self._get_users_for_group(group_item, options), req_options))
```

Comment:
Yeah, this was before we spoke the second time.

I'll need to do more rework to update all the old tests to actually test add/remove user properly. Stay tuned.